Octokit.Reactive/Clients/IObservableIssuesEventsClient.cs

```diff
@@ -20,7 +20,7 @@ public interface IObservableIssuesEventsClient
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
         /// <param name="number">The issue number</param>
-        IObservable<EventInfo> GetAllForIssue(string owner, string name, int number);
+        IObservable<EventInfo> GetAllForIssue(string owner, string name, long number);
```

Comment:
Same thing for this file - `number` here is the issue number itself, no the `EventInfo` number that is being deserialized.